cmd/wallet-sdk-gomobile/openid4ci/interaction.go

```diff
@@ -73,9 +73,22 @@ func (c *ClientConfig) DisableVCProofChecks() {
 	c.disableVCProofChecks = true
 }
 
+// AddHeaders adds the given HTTP headers to all REST calls made to the issuer during the OpenID4CI flow.
+func (c *ClientConfig) AddHeaders(headers *api.Headers) {
+	if c.httpClient == nil {
```

Comment:
The same as in resolver. better to store headers and httpClientSkipTLSVerify in ClientConfig, and create httpClient on config unwrap

Reply:
Done

Reply:
I also made a small change to the `httpClient` file to make it cleaner to set the `DisableTLSVerification` flag

Reply:
looks good